### PR TITLE
Add plugin metadata documentation and example

### PR DESCRIPTION
**Pull request by Kuuuube**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,4 +15,39 @@ Plugin-Repository/Repository/{Version}/{Owner}/{Repository}/{Name}.json
...
+| CompressionFormat                    | The compression format used by the compressed archive in the DownloadUrl field. `zip` is preferred.                                                                                                                    |
+| SHA256                               | The SHA265 hash of the compressed archive in the DownloadUrl field.                                                                                                                                                    |
+| (Optional) WikiUrl                   | A link to the plugin's documentation.                                                                                                                                                                                  |
+| LicenseIdentifier                    | The plugin's license name(s).                                                                                                                                                                                          |
+
+Example plugin metadata:
+
+```
```
Comment (X9VoiD) [inline README.md:39]:
```suggestion
```json
```

File: README.md [hunk 2]
```diff
@@ -15,4 +15,39 @@ Plugin-Repository/Repository/{Version}/{Owner}/{Repository}/{Name}.json
...
+| (Optional) MaxSupportedDriverVersion | The maximum OpenTabletDriver version the plugin functions on in four-part version number format. This should only be included in cases where a minor OpenTabletDriver version introduced breaking changes to a plugin. |
+| RepositoryUrl                        | The git repository url containing the plugin's source code.                                                                                                                                                            |
+| DownloadUrl                          | The direct download url for the plugin's release files in a compressed archive format. The plugin's dll files must be contained in the top-level directory of the compressed archive.                                  |
+| CompressionFormat                    | The compression format used by the compressed archive in the DownloadUrl field. `zip` is preferred.                                                                                                                    |
```
Comment (X9VoiD) [inline README.md:32]:
```suggestion
| CompressionFormat                    | The compression format used by the compressed archive in the DownloadUrl field. Only `zip` is supported right now.                                                                                                                    |
```

File: README.md [hunk 3]
```diff
@@ -15,4 +15,39 @@ Plugin-Repository/Repository/{Version}/{Owner}/{Repository}/{Name}.json
...
+| SHA256                               | The SHA265 hash of the compressed archive in the DownloadUrl field.                                                                                                                                                    |
+| (Optional) WikiUrl                   | A link to the plugin's documentation.                                                                                                                                                                                  |
+| LicenseIdentifier                    | The plugin's license name(s).                                                                                                                                                                                          |
```
Comment (X9VoiD) [inline README.md:35]:
```suggestion
| LicenseIdentifier                    | The plugin's SPDX license identifier(s), separated by comma.                                                                                                                                                                                          |
```

**Review by X9VoiD — CHANGES_REQUESTED** [review]
LGTM, just needs some minor improvements.

File: README.md (hunk 2 above)
Comment (jamesbt365) [reply README.md:32]: "currently" instead?

Comment (X9VoiD) [reply README.md:32]: Only `zip` is currently supported?

Comment (jamesbt365) [reply README.md:32]: Yeah seems better imo

Comment (Kuuuube) [reply README.md]: Changed to "Currently, `zip` is the only supported value."

**Pull request merged**